client/src/models/enums/contactQuestioningExcelFields.ts

```diff
@@ -0,0 +1,34 @@
+import InteractedContact from 'models/InteractedContact';
+
+export enum booleanAnswers {
+    TRUE = 'כן',
+    FALSE = 'לא'
+};
+
+export const ContactedPersonFieldMapper: ContactedPersonExcel= {
+    firstName: 'שם פרטי',
+    lastName: 'שם משפחה',
+    phoneNumber: 'מספר טלפון',
+    additionalPhoneNumber: 'טלפון משני',
+    birthDate: 'תאריך לידה',
+    gender: 'מגדר',
+    identificationNumber: 'ת.ז',
+    identificationType: 'סוג ת.ז',
+    contactDate: 'תאריך חשיפה',
+    contactType: 'סוג מגע',
+    extraInfo: 'פירוט נוסף על אופי המגע',
+    cantReachContact: 'אין מענה',
+    relationship: 'קשר',
+    familyRelationship: 'קרבה משפחתית',
+    occupation: 'האם עוסק באחד מן התחומים הבאים',
+    doesHaveBackgroundDiseases: 'האם סובל ממחלות רקע',
+    contactedPersonCity: 'יישוב השהייה בבידוד',
+    doesFeelGood: 'האם חש בטוב',
+    doesNeedHelpInIsolation: 'האם נדרש סיוע עבור מקום בידוד',
+    repeatingOccuranceWithConfirmed: 'מפגש חוזר עם המאומת',
+    doesLiveWithConfirmed: 'האם חי באותו הבית עם המאומת',
+    doesWorkWithCrowd: 'עבודה עם קהל במסגרת העבודה',
+    doesNeedIsolation: 'הקמת דיווח בידוד',
+};
+
+export type ContactedPersonExcel =  { [K in keyof Omit<Omit<InteractedContact, 'id'>,'contactEvent'>]: string };
```

Comment:
https://www.typescriptlang.org/docs/handbook/utility-types.html#excludetype-excludedunion

Maybe you can use Exclude instead of 2 omits

Reply:
https://www.typescriptlang.org/docs/handbook/utility-types.html#picktype-keys

Or maybe use Omit once with 2 keys like the Pick